diff -up java-1.8.0-openjdk-1.8.0.352.b08-1.fc38.x86_64/openjdk/jdk/src/solaris/native/sun/nio/fs/LinuxNativeDispatcher.c.c99 java-1.8.0-openjdk-1.8.0.352.b08-1.fc38.x86_64/openjdk/jdk/src/solaris/native/sun/nio/fs/LinuxNativeDispatcher.c
--- java-1.8.0-openjdk-1.8.0.352.b08-1.fc38.x86_64/openjdk/jdk/src/solaris/native/sun/nio/fs/LinuxNativeDispatcher.c.c99	2022-12-11 22:23:17.729952935 +0100
+++ java-1.8.0-openjdk-1.8.0.352.b08-1.fc38.x86_64/openjdk/jdk/src/solaris/native/sun/nio/fs/LinuxNativeDispatcher.c	2022-12-11 22:23:44.801668107 +0100
@@ -29,6 +29,7 @@
 #include "jlong.h"
 
 #include <stdio.h>
+#include <stdlib.h>
 #include <string.h>
 #include <dlfcn.h>
 #include <errno.h>
