diff -r 9d6eb2757167 make/defs.make
--- openjdk/hotspot/make/defs.make	Wed Feb 11 18:56:26 2015 -0800
+++ openjdk/hotspot/make/defs.make	Tue Feb 17 16:39:22 2015 +0000
@@ -317,6 +317,13 @@
   LIBARCH/arm     = arm
   LIBARCH/zero    = $(ZERO_LIBARCH)
 
+  # Override LIBARCH for ppc64le
+  ifeq ($(ARCH), ppc64)
+    ifeq ($(OPENJDK_TARGET_CPU_ENDIAN), little)
+      LIBARCH = ppc64le
+    endif
+  endif
+
   LP64_ARCH = sparcv9 amd64 ia64 ppc64 zero
 endif
 
diff -r 9d6eb2757167 src/os/linux/vm/os_linux.cpp
--- openjdk/hotspot/src/os/linux/vm/os_linux.cpp	Wed Feb 11 18:56:26 2015 -0800
+++ openjdk/hotspot/src/os/linux/vm/os_linux.cpp	Tue Feb 17 16:39:22 2015 +0000
@@ -265,7 +265,11 @@
 #elif defined(PPC32)
 static char cpu_arch[] = "ppc";
 #elif defined(PPC64)
+#if defined(VM_LITTLE_ENDIAN)
+static char cpu_arch[] = "ppc64le";
+#else
 static char cpu_arch[] = "ppc64";
+#endif
 #elif defined(SPARC)
 #  ifdef _LP64
 static char cpu_arch[] = "sparcv9";
